gdal/frmts/grib/gribcreatecopy.cpp

```diff
@@ -283,10 +295,36 @@ bool GRIB2Section3Writer::WriteGeographic()
 
     WriteEllipsoidAndRasterSize();
 
-    if( dfLLX < 0 )
+    CPLDebug("GRIB", "Geotransform: %lf %lf %lf", adfGeoTransform[0], adfGeoTransform[1], adfGeoTransform[2]);
```

Comment:
```suggestion
```

Reply:
In fact this is a stray debug message, I will just remove it